lambdas/handlers/document_manifest_job_handler.py

```diff
@@ -14,9 +17,48 @@
 logger = LoggingService(__name__)
 
 
-@set_request_context_for_logging
 @validate_patient_id
 @validate_document_type
+def create_manifest_job(event, context):
+    logger.info("Starting document manifest process")
+    nhs_number = event["queryStringParameters"]["patientId"]
+    document_type = extract_document_type_to_enum(
+        event["queryStringParameters"]["docType"]
+    )
+    document_references = event["multiValueQueryStringParameters"].get("docReferences")
+
+    request_context.patient_nhs_no = nhs_number
+
+    document_manifest_service = DocumentManifestJobService()
+    response = document_manifest_service.create_document_manifest_job(
+        nhs_number, document_type, document_references
+    )
+
+    return ApiGatewayResponse(
+        200, json.dumps({"jobId": response}), "POST"
+    ).create_api_gateway_response()
+
+
+@validate_manifest_job_id
+def get_manifest_job(event, context):
+    logger.info("Retrieving document manifest")
+
+    job_id = event["queryStringParameters"]["jobId"]
+
+    document_manifest_service = DocumentManifestJobService()
+    response = document_manifest_service.create_document_manifest_presigned_url(job_id)
+
+    logger.audit_splunk_info(
+        "User has downloaded Lloyd George records",
+        {"Result": "Successful download"},
+    )
```

Comment:
probably better to change this log, as we can't tell whether the user is getting a completed response with url. Even for case of status = Pending it logs this line.